doc/user/sql/create-source/_index.md

```diff
@@ -20,7 +20,7 @@ Materialize can connect to many different external sources of data, each with
 their own requirements. For details about creating sources, view the
 documentation for the type of data you are trying to load into Materialize:
 
-- [Avro over Kafka](./avro-source)
+- [Avro, raw bytes, or raw text over Kafka](./avro-source)
```

Comment:
This should be broken out into a new doc that's `Raw bytes or text over Kafka`; if Avro data also allows the new upsert envelope, the upsert details can remain in there, but the new doc should not have Avro details.

Reply:
Let me know if you'd like me to take care of this for you; can do it on your fork or just create a new PR.